### PR TITLE
Fix linked data properties for Google SEO

### LINKED ISSUE #6840: Invalid linked data properties for live radio pages
**Describe the bug**
Invalid linked data properties for live radio pages - `thumbnailUrl` and `publisher`.

Screenshot 2020-06-09 at 09 12 13

**To Reproduce**
Steps to reproduce the behaviour:

1. Go to https://search.google.com/structured-data/testing-tool
2. Enter `https://www.bbc.com/korean/bbc_korean_radio/liveradio`

Observe invalid property warning for `thumbnailUrl` and `publisher`.

**Expected behaviour**

**Screenshots**

**Desktop (please complete the following information):**

**Smartphone (please complete the following information):**

**Testing notes**

- [ ] This bug fix is expected to need manual testing.

**Additional context**

### PR DESCRIPTION
**Pull request by pharingee**
Resolves #6840

**Overall change:**
Fix a bug in Google linked data that used unsupported linked data tags

**Code changes:**
- Add a helper to clean linked data
- Write tests

---

- [x] I have assigned myself to this PR and the corresponding issues
- [x] I have added labels to this PR for the relevant pod(s) affected by these changes
- [x] I have assigned this PR to the Simorgh project

**Testing:**

- [x] Automated (jest and/or cypress) tests added (for new features) or updated (for existing features)
- [ ] If necessary, I have run the local E2E non-smoke tests relevant to my changes (`CYPRESS_APP_ENV=local CYPRESS_SMOKE=false npm run test:e2e:interactive`)
- [x] This PR requires manual testing

### CODE REVIEW

File: src/app/containers/LinkedData/helpers.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
+/* eslint-disable import/prefer-default-export */
+export const cleanLinkedData = linkedData => {
```
Comment (karinathomasbbc) [inline src/app/containers/LinkedData/helpers.js]: I prefer `utilities` to `helpers` - could we rename this file?

Comment (karinathomasbbc) [inline src/app/containers/LinkedData/helpers.js]: Alternatively, could we just have a utilities folder, and have `cleanLinkedData` as a separate file + test?

Comment (jroebu14) [reply src/app/containers/LinkedData/helpers.js]: Seems like an overly complex way to conditionally add 2 properties

Comment (jroebu14) [reply src/app/containers/LinkedData/helpers.js]: inside of https://github.com/bbc/simorgh/pull/6901/files#diff-0e25abbcfed44609b2085dac70092114R62

can you not just do?

```js
  const linkedData = {
    '@type': type,
    url: canonicalNonUkLink,
    image,
    mainEntityOfPage,
...(type === 'WebPage' && { publisher, thumbnailUrl: defaultImage }),
```

Comment (pharingee) [reply src/app/containers/LinkedData/helpers.js]: I might be guilty here of over-optimizing, but it seems to be that there could be specific changes down the line. I guess I could go ahead and include it in the main file

**Review by karinathomasbbc — APPROVED** [review]
Looks good, thanks for this 👍

**Review by jroebu14 — APPROVED** [review]

File: src/app/containers/LinkedData/helpers.js (hunk 1 above)
Comment (jroebu14) [reply src/app/containers/LinkedData/helpers.js]: Thanks for the changes. We could consider something like that in the future if we need it though

**Pull request merged**